core/src/main/java/com/beepiz/bluetooth/gattcoroutines/experimental/GattConnection.kt

```diff
@@ -96,7 +95,7 @@ interface GattConnection {
      * [this video](https://youtu.be/qx55Sa8UZAQ?t=28m30s) is **4 on Android 4.3**,
      * 7 on Android 4.4, and 15 on Android 5.0+.
      */
-    fun setCharacteristicNotificationsEnabled(characteristic: BGC, enable: Boolean)
+    fun setCharacteristicNotificationsEnabled(characteristic: BGC, enable: Boolean, type: ByteArray)
```

Comment:
This change is beyond the scope of Kotlin 1.3 update.
This should be discussed in #5 before a PR is worked on.

Reply:
Sure. will do it. Thanks for your comments.